pkg/cmd/server/server.go

```diff
@@ -72,10 +71,10 @@ import (
 
 	ctrl "sigs.k8s.io/controller-runtime"
 
-	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	"github.com/vmware-tanzu/velero/internal/util/managercontroller"
+	"github.com/vmware-tanzu/velero/internal/velero"
```

Comment:
Why?

Reply:
Unless I'm misunderstanding, this would be true but only for this file given the scope of an alias. Right? I'm not sure this is worth it. Maybe I'm misunderstanding. And on another hand and by this logic: are you proposing we alias every package that goes under `internal`?

Reply:
Considering the strong conventions, imposed by the go tooling, I am OK with not aliasing the imports.
https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface